client/src/main/java/model/Fleet.java

```diff
@@ -8,38 +8,39 @@
     private final List<Mast> masts;
     private final Sea sea;
     private final List<Ship> fleet;
-    private Ship lastShip;
+    private Ship shipThatIsBuild;
 
     public Fleet(Sea sea) {
         this.sea = sea;
         masts = new ArrayList<>();
         fleet = new ArrayList<>();
     }
 
-    private void addFirstMastToShip(Mast mast) {
+    private void addFirstMastToFleet(Mast mast) {
         masts.add(mast);
     }
 
-    public void startToBuilOneShip(Mast mast) {
-        addFirstMastToShip(mast);
-        Ship ship = new Ship(mast);
-        lastShip = ship;
+    public void startToBuildOneShip(Mast mast, int shipLength) {
```

Comment:
only start build or it build? :)

Reply:
Only start , rest of building is somewhere else